README.md

```diff
@@ -67,26 +71,33 @@ This resource has the following actions:
 ### Attributes ###
 This resource has the following attributes:
 
-* `storage_system` string. Required, name_attibute. IP address of the storage system.
+* `storage_system` string. Required, name_attribute. IP address of the storage system.
```

Comment:
@mattray 'storage_system' is correct since we are using that inside recipe for e.g.

netapp_e_host_group node['netapp']['host_group']['name'] do
  **storage_system** node['netapp']['storage_system_ip']
  hosts node['netapp']['host_group']['hosts']
  action :create
end

We just used storage_system_ip to set default attribute. So according to me we don't have to change readme.